packages/scss/src/components/inputs/textfield/_textfield.scss

```diff
@@ -78,62 +92,37 @@
 	&:hover {
 		background: _theme("commons.background.dark");
 	}
-}
-
-.textfield-messages {
-	line-height: 1.3;
-	margin-top: .3rem;
-}
-
-.textfield-messages-error {
-	color: _color("error");
-	font-size: _component("textfield.message.font-size");
-	margin: 0;
-}
-
-.textfield-messages-warning {
-	color: _color("warning");
-	font-size: _component("textfield.message.font-size");
-	margin: 0;
-}
 
-.textfield-messages-helper {
-	color: _color("text.light");
-	font-size: _component("textfield.message.font-size");
-	margin: 0;
+	&.is-focus {
+		background: _theme("commons.background.dark");
+	}
 }
 
-.textfield-suffix {
```

Comment:
You should not remove `.textfield-suffix` declaration. Its `right` and `top` position might use those defined in `_textfield.compact.scss`. We should use variables here for positionning as it should be the same for the ::after of `.mod-autocomplete` and `.mod-select`

Reply:
✔️